buildenv/jenkins/variables/defaults.yml

```diff
@@ -431,6 +431,47 @@ x86-64_windows:
       12: 'PATH+TOOLS=/cygdrive/c/openjdk/LLVM64/bin:/cygdrive/c/openjdk/nasm-2.13.03'
       next: 'PATH+TOOLS=/cygdrive/c/openjdk/LLVM64/bin:/cygdrive/c/openjdk/nasm-2.13.03'
 #========================================#
+# Windows x86 64bits Large Heap
+#========================================#
+x86-64_windows:
+  boot_jdk:
+    8: '/cygdrive/c/openjdk/jdk7'
+    9: '/cygdrive/c/openjdk/jdk8'
+    10: '/cygdrive/c/openjdk/jdk9'
+    11: '/cygdrive/c/openjdk/jdk11'
+    12: '/cygdrive/c/openjdk/jdk11'
+    next: '/cygdrive/c/openjdk/jdk12'
+  release:
+    8: 'windows-x86_64-normal-server-release'
+    9: 'windows-x86_64-normal-server-release'
+    10: 'windows-x86_64-normal-server-release'
+    11: 'windows-x86_64-normal-server-release'
+    12: 'windows-x86_64-server-release'
+    next: 'windows-x86_64-server-release'
+  freemarker: '/cygdrive/c/openjdk/freemarker.jar'
+  openjdk_reference_repo: '/cygdrive/c/openjdk/openjdk_cache'
+  extra_configure_options:
+    8: '--with-freetype-include=/cygdrive/c/openjdk/freetype-2.5.3/include --with-freetype-lib=/cygdrive/c/openjdk/freetype-2.5.3/lib64 --disable-ccache --with-openssl=/cygdrive/c/openjdk/OpenSSL-1.1.1-x86_64 --enable-openssl-bundling --with-noncompressedrefs'
+    9: '--with-freetype-src=/cygdrive/c/openjdk/freetype-2.5.3 --with-toolchain-version=2013 --disable-ccache --with-noncompressedrefs'
```

Comment:
9 and 10 can be removed.
Please put the `--with-noncompressedrefs` at the beginning of the line, similarly to the other plats that specify this.

Reply:
Done